external/test-runtime/project.json

```diff
@@ -19,7 +20,12 @@
     "System.Net.TestData": "1.0.0-prerelease"
   },
   "frameworks": {
-    "netstandard2.0": {}
+    "netstandard2.0": {},
+    "net463": {
+      "dependencies": {
+        "Microsoft.TargetingPack.NETFramework.v4.6.1": "1.0.1"
```

Comment:
Why do we need this here? The targeting pack should come from https://github.com/dotnet/corefx/tree/master/external/netfx